client-v2/src/components/FrontsCAPIInterface/SearchInput.js

```diff
@@ -6,15 +6,19 @@ import styled from 'styled-components';
 import SearchQuery from '../CAPI/SearchQuery';
 import ScrollContainer from '../ScrollContainer';
 import TextInput from '../TextInput';
+import CAPITagInput from '../FrontsCAPIInterface/TagInput';
 
 type FrontsCAPISearchInputProps = {
   children: *,
-  additionalFixedContent?: React.ComponentType<any>
+  additionalFixedContent?: React.ComponentType<any>,
+  displaySearchFilters: boolean
 };
 
 type FrontsCAPISearchInputState = {
   q: ?string,
-  tag: ?string
+  tags: Array<string>,
```

Comment:
Is this an array of tag ids?

Reply:
Yes. We use this array to query capi for content with these tags.